packages/app/src/specs/InlineSpecListTree.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     v-bind="containerProps"
-    class="pt-8px specs-list-container overflow-y-auto overflow-x-hidden"
+    class="pt-8px specs-list-container overflow-hidden"
```

Comment:
Since the scrolling in the specs list takes place within a virtual list, the overflow-x value of `auto` was causing a weird empty space at the bottom of the parent, to do with how the `vh` works in the `specs-list-container` height calculation below.